hooks/shapediver/viewer/interaction/useNodeInteractionData.ts

```diff
@@ -0,0 +1,125 @@
+import { GeometryData, IOutputApi, ITreeNode } from "@shapediver/viewer";
+import { useCallback, useEffect } from "react";
+import { IInteractionData, InteractionData } from "@shapediver/viewer.features.interaction";
+import { useOutputNode } from "../useOutputNode";
+
+const nodesWithInteractionData: { [key: string]: { node: ITreeNode, data: IInteractionData } } = {};
```

Comment:
@MajorMeerkatThe3rd let's document what exactly we store here (what the key values are)